src/careamics/cli/conf.py

```diff
@@ -0,0 +1,404 @@
+"""Configuration building convinience functions for the CAREamics CLI."""
+
+import os
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Tuple
+
+import click
+import typer
+import yaml
+from typing_extensions import Annotated
+
+# import hydra
+from ..config import (
+    Configuration,
+    create_care_configuration,
+    create_n2n_configuration,
+    create_n2v_configuration,
+)
+
+WORK_DIR = Path(os.getcwd())
```

Comment:
Not very important, but I would use `pathlib` rather than `os` whenever we can. It returns directly a `Path` and is more modern.

here `Path.cwd()` would do the trick!

Reply:
fixed